### ISSUE TITLE
Add extension variables and parameters

### ISSUE THREAD

**Issue by guillett** [openfisca/legislation-explorer#87]

**Comment by MattiSG** [comment]
This one is not obvious: which extensions should be loaded? Maybe it is up to each extension vendor to set up their own instance of the legislation explorer.

@guillett could you please provide us with some more insight into your use case? Which extensions would you need, and why is their documentation not sufficient?

**Comment by Morendil** [comment]
Yes, for instance https://fr.openfisca.org/legislation/ makes sense as "the explorer for France" but it's harder to infer from the subdomain a canonical set of extensions that it should be loading.

It would make sense (for instance) for Mes Aides to have access to an explorer instance that eases debugging for the situations computed on behalf of Mes Aides users. It might be a good idea to make this a test bed for #143, reframing the definition of the goal as "Help Mes Aides host an instance of the Legislation Explorer that shows variable from all extensions configured to be loaded". Possibly at https://mes-aides.gouv.fr/legislation/ See what's involved in doing that, and smooth over the more significant frictions we find.

@guillett WDYT ?
